src/components/Home/Home.css

```diff
@@ -0,0 +1,111 @@
+.imageScore {
+  width: 50%;
+}
+
+.imageScore img {
+  width: 100%;
+}
+
+.games {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  align-items: center;
+  justify-content: center;
+  color: lightgray;
+}
+
+.title {
+  color: white;
+}
+
+.arrow-right {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.arrow-btn {
+  background: none;
+  border: none;
+}
+
+.arrow {
+  font-size: 1.5rem;
+  color: white;
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+}
+
+.game {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 0.5rem;
```

Comment:
- [ ] Kindly make sure you improve your code and make it DRY(don't repeat yourself). I can see the  `display: flex;` being used many times but you can extra it and put it in one class and be using it to HTML tags. this will make your code clean and clear.